pkg/common/errors.go

```diff
@@ -2,27 +2,37 @@ package common
 
 import (
 	"net/http"
+	"reflect"
 
 	"github.com/nuclio/errors"
 	"github.com/nuclio/nuclio-sdk-go"
 )
 
 func ResolveErrorStatusCodeOrDefault(err error, defaultStatusCode int) int {
```

Comment:
👏 

Some ideas for unit-testings:

- `err.wrap([err, err, err_with_status])` -> returns `err_with status`
- `err.wrap([err, err_with_status, err])` -> returns `err_with status`
- `err.wrap([err_with_status, err, err])` -> returns `err_with status`

- `err.wrap([err_with_status, err_with_status])` -> return first `err_with_status`
- `err` -> return internal
- `nil` -> return default